src/backends/pytorch/libtorch_backend.cc

```diff
@@ -244,13 +277,19 @@ Status
 LibTorchBackend::Context::ValidateOutputs(
     const ::google::protobuf::RepeatedPtrField<ModelOutput>& ios)
 {
+  std::string deliminator = "__";
+
   for (const auto& io : ios) {
     const auto pr = ConvertDataTypeToTorchType(io.data_type());
     if (!pr.first) {
       return Status(
           RequestStatusCode::INTERNAL,
           "unsupported datatype " + DataType_Name(io.data_type()) +
               " for output '" + io.name() + "' for model '" + name_ + "'");
+    } else {
+      const std::string& name = io.name();
+      int op_index = std::atoi(name.substr(name.find(deliminator) + 2).c_str());
+      io_index_map_[name] = op_index;
```

Comment:
I don't think you can use the same map for both input and output. What if I have an input with the same name as an output?

Reply:
Right. Will fix that